addons/binding/org.openhab.binding.evohome/README.md

```diff
@@ -0,0 +1,119 @@
+# evohome Binding
+
+This binding integrates the Honeywell evohome system. It uses your Honeywell Total Connect Comfort account to access your locations and heating zones.
```

Comment:
Please add a line break after every sentence (it won't change the rendering, but will make diffing/reviewing easier).